.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  "env": {
+    "es2021": true,
+    "browser": true
+  },
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "error",
+    "no-console": 1
```

Comment:
We can keep pattern in values, I prefer string values. What do you think?

Reply:
This config doesn't accept strings as values, I don't know why, prob because it's an enum.

Reply:
Sad 😢